lib/tls_test_kit/tls_version_test.rb

```diff
@@ -78,30 +87,41 @@ def version_required?(version)
         begin
           http.request_get(uri)
           if self.class.version_forbidden? version
-            add_message('error', "Server incorrectly allowed #{version_string} connection.")
+            message =
+              "#{url} allowed #{version_string} connection even though #{version_string} connections should be denied."
```

Comment:
Can we add this language?  (rubocop may need reformatting)

```suggestion
              "#{url} accepted #{version_string} connection even though #{version_string} connections should be denied.  Manual verification that content is not sent over this connection is required."
```

Reply:
This is a really general purpose test, and I'm reluctant to add language to it to say that a tester is required to do something. I would prefer the suggested language from below:
> The system may deny content from being sent over this connection, but this must be manually verified.

Reply:
Sounds good 👍